superset-frontend/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -289,9 +290,13 @@ class TabbedSqlEditors extends React.PureComponent {
       const title = (
         <>
           {qe.title} <TabStatusIcon tabState={state} />{' '}
-          <span className="close" onClick={() => this.removeQueryEditor(qe)}>
-            {'×'}
-          </span>
+          <Icon
+            name="cancel-x"
+            height="16px"
+            width="16px"
+            viewBox="4 4 16 16"
```

Comment:
is this required because the viewbox wasn't set correctly on the original icon? if so, maybe we should change the SVG, since it's not used anywhere else yet